docs/jackson.md

```diff
@@ -0,0 +1,190 @@
+# Serializing Eclipse Collections with Jackson
```

Comment:
Include a Copyright Header similar the [ref guide.](https://raw.githubusercontent.com/eclipse/eclipse-collections/master/docs/guide.md)